### DISCUSSION TITLE
Unable to setup Security Onion in VirtualBox 7.0 on Windows 10. Blue screen of death, blocking state, forwarding state and IPTables Dropped

### DISCUSSION THREAD

**Discussion by pn3m0n1c** [Security-Onion-Solutions/securityonion#12931] [Unsupported Versions]
Hello! 

I am having an issue trying to setup Security Onion on Virtual Box. I can finish the installation correctly, having the installation succesful final message. 

My VirtualBox config is:
- VM with only one Bridge Adapter
- 4GB RAM (Also tried with 6GB, 8GB RAM, same results) 
- 100GB ROM (Also tried with 125, 120 and 130GB ROM) 

I create the VM empty, select it and then select the security onion ISO to load it, _for some reason, if I load the ISO and let VM find out the OS, the installation got an error at the end, tried 3 times to be sure and 3 times loading the ISO previously got the same error at the end of installation_, so the only way to finish the installation is to load the ISO after VM has started.

This is my installation process:

The first setup runs smoothly with no errors, then this config steps show up:

![imagen](https://github.com/Security-Onion-Solutions/securityonion/assets/92324278/a405a4a9-d586-432b-8d24-e565aa2924c3)

![imagen](https://github.com/Security-Onion-Solutions/securityonion/assets/92324278/eb817aa9-970d-4dc7-a987-958e82d4ef00)

![imagen](https://github.com/Security-Onion-Solutions/securityonion/assets/92324278/4d374138-af7e-47d2-b3fb-b00763b60517)

![imagen](https://github.com/Security-Onion-Solutions/securityonion/assets/92324278/72371618-5182-44d7-a086-e3023be4d615)

![imagen](https://github.com/Security-Onion-Solutions/securityonion/assets/92324278/6619db7d-0d60-441c-ad0c-bf6728e7100e)

![imagen](https://github.com/Security-Onion-Solutions/securityonion/assets/92324278/c4aaf57f-1071-4970-b9dd-e1f27a96c619)

![imagen](https://github.com/Security-Onion-Solutions/securityonion/assets/92324278/4e8cdfd6-ad63-43ae-994c-0e5e2323fb2f)

![imagen](https://github.com/Security-Onion-Solutions/securityonion/assets/92324278/5b9ce0e2-1829-45a1-87b7-4287d4ecd34a)

![imagen](https://github.com/Security-Onion-Solutions/securityonion/assets/92324278/307d3815-2d1b-4b97-832c-f486d8893fb5)

![imagen](https://github.com/Security-Onion-Solutions/securityonion/assets/92324278/e134678e-efc2-4473-ba8c-e60f804a3560)

![imagen](https://github.com/Security-Onion-Solutions/securityonion/assets/92324278/64f1c45f-e64f-48d9-a1a3-f22befd3f1d0)

Above I set the IP to my local network address which is 192.168.0.0/24 (I tried with /24 and without it, same result for both options).

![imagen](https://github.com/Security-Onion-Solutions/securityonion/assets/92324278/b6be8fd9-0795-4b8f-94b8-bc34209d9aef)

![imagen](https://github.com/Security-Onion-Solutions/securityonion/assets/92324278/062e48ec-4159-407a-882e-a14a51d90480)

After this, the installation begins and ends succesfully. 

But when I try to access the web Interface, I have even got two blue screen errors with a virtual box error code (it came up very fast and I couldn't capture it to put it here, but I could read virtual box). I have not been able to access the web interface yet.

I have reinstalled like 7-8 times, tried by changing network card settings like promiscuous mode, adapter type... but nothing worked. These logs appear in the terminal of the virtual machine after a while:

![imagen](https://github.com/Security-Onion-Solutions/securityonion/assets/92324278/f2369f07-5eb8-4b89-b39c-2f02ce1ef48f)

I even got a blue screen error with a virtual box error code, (it came up very fast and I couldn't capture it to put it here, but I could read virtual box). I have reinstalled by changing network card settings like promiscuous mode, adapter type... but nothing worked. These logs appear in the terminal of the virtual machine after a while.
 
They even appear in the user input and password field, overwriting the fields:

![imagen](https://github.com/Security-Onion-Solutions/securityonion/assets/92324278/46d7a820-822a-424a-b110-4a8f5f0c9d8a)

Also, the sudo so-status output is always the same:

![imagen](https://github.com/Security-Onion-Solutions/securityonion/assets/92324278/e613e6ae-9b7c-454c-8507-780d90b165ac)

For more information, I have noticed that this error appears during the boot proccess:

![Screenshot_11](https://github.com/Security-Onion-Solutions/securityonion/assets/92324278/2d7d76b0-30cc-4d73-9c5a-5ade2a207cf8)

I tried to `chmod +x` the `/etc/rc.d` file with `sudo` privileges. The error message dissapears, but nothing changes, same IPTables and network errors appears and I am unable to access the web interface or load the SO.

I hope someone can help me! 

Thank you so much for your attention.

**Comment by dougburks** [comment]
Here are some thing you can try:
- Make sure that you're not oversubscribing your host machine. The host needs enough CPU, RAM, and storage to run the host operating system and any virtualized operating systems. If you are running any other VMs or any other applications, try shutting them down.
- Try running hardware diagnostics on the host machine to rule out any possible hardware issues.
- Try downloading the 2.4.60 ISO. Make sure you verify the ISO image.
- Try different virtualization software like VMware.
- Try a different host machine.
- Try installing directly on a machine without virtualization.
- Consider upgrading the host machine (increasing RAM, moving to SSD, etc.)
- Review the documentation at https://docs.securityonion.net/en/2.4/hardware.html
